styles/AdvancedEditor.css

```diff
@@ -715,3 +730,7 @@ Next levels are shifted 'up' by 1
 .citation-wrapper:hover .track-changes-review {
   display: inline-block;
 }
+[data-cy="author-container"] table .gonusi {
+  font-weight: 400;
+  padding-right: 10pt;
+}
```

Comment:
I can't get this selector!

Reply:
![image](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/93189633/a7131434-916b-4e76-8835-ae011a0a2246)
![image](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/93189633/506b3cb8-965b-42b6-93d9-87596aeaefa1)

Reply:
will this is styled in the [style-guide](https://github.com/Atypon-OpenSource/manuscripts-style-guide/blob/bfde3f911d7b176ab6a3f9b2747df50d115ccc03/src/components/AuthorsContainer.tsx#L50), and it's out of the scope of the ticket!

Reply:
@asouqi  removed..please check.